peps/pep-0731.rst

```diff
@@ -0,0 +1,181 @@
+PEP: 731
+Title: C API Working Group Charter
+Author: Guido van Rossum <guido@python.org>,
+        Petr Viktorin <encukou@gmail.com>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Irit Katriel <irit@python.org>
+Status: Draft
+Type: Process
+Topic: Typing
+Created: 11-Oct-2023
+Python-Version: 3.13
+
+Abstract
+========
+
+This PEP proposes to establish the C API Working Group:
+a small committee of Python core developers responsible for
+overseeing and coordinating the development and maintenance of the Python C API.
+
+The working group will maintain
+documentation, test suites and tooling related to Python's C API.
+It is the deciding body for changes to the C API,
```

Comment:
This is a strong statement and it sounds different from the "Relationship with the Steering Council" section below, which states that the SC remains the decision-making authority on relevant PEPs.

I assume you've talked about this with the SC members present in Brno?

Reply:
Honestly I haven't yet, and in the rush to get something out all of us missed this. I'll think of better words.

Reply:
I solved this by inserting "As delegated by the Steering Council".

Reply:
(And the SC members present in Brno are very aware of our plans.)